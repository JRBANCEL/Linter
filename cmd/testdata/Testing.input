package test

import (
	"testing"
)

func TestFatatfTwoArgsSpace(t *testing.T) {
	arg := "test"
	t.Fatalf("YOLO: %v", arg)
}

func TestFatatfTwoArgsNoSpace(t *testing.T) {
	arg := "test"
	t.Fatalf("YOLO:%v", arg)
}

func TestFatalfThreeArgs(t *testing.T) {
	arg0 := "test"
	arg1 := "yolo"
	t.Fatalf("YOLO %v and %v", arg0, arg1)
}

func BenchmarkInfofTwoArgsSpace(b *testing.B) {
	arg := "test"
	b.Infof("YOLO: %v", arg)
}

func BenchmarkInfofTwoArgsNoSpace(b *testing.B) {
	arg := "test"
	b.Infof("YOLO:%v", arg)
}

func BenchmarkInfofThreeArgs(b *testing.B) {
	arg0 := "test"
	arg1 := "yolo"
	b.Infof("YOLO %v and %v", arg0, arg1)
}