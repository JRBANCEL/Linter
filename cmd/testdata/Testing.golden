package test

import (
	"testing"
)

func TestFatatfTwoArgsSpace(t *testing.T) {
	arg := "test"
	t.Fatal("YOLO:", arg)
}

func TestFatatfTwoArgsNoSpace(t *testing.T) {
	arg := "test"
	t.Fatal("YOLO:", arg)
}

func TestFatalfThreeArgs(t *testing.T) {
	arg0 := "test"
	arg1 := "yolo"
	t.Fatalf("YOLO %v and %v", arg0, arg1)
}

func BenchmarkInfofTwoArgsSpace(b *testing.B) {
	arg := "test"
	b.Info("YOLO:", arg)
}

func BenchmarkInfofTwoArgsNoSpace(b *testing.B) {
	arg := "test"
	b.Info("YOLO:", arg)
}

func BenchmarkInfofThreeArgs(b *testing.B) {
	arg0 := "test"
	arg1 := "yolo"
	b.Infof("YOLO %v and %v", arg0, arg1)
}